crates/api_crud/src/community/update.rs

```diff
@@ -61,6 +61,11 @@ impl PerformCrud for EditCommunity {
       nsfw: data.nsfw,
       posting_restricted_to_mods: data.posting_restricted_to_mods,
       updated: Some(naive_now()),
+      is_default_community: if local_user_view.person.admin {
+        data.is_default_community
+      } else {
+        read_community.is_default_community
+      },
```

Comment:
Should be done like this: before the form, add a check like 
```
if data.is_default_community.is_some() {
    is_admin(&local_user_view)?;
}
```

And in the form, just put `is_default_community: data.is_default_community`.

Reply:
In this case, data.is_default_community is just a bool, so its always valid. Also it is a & non-mutable type, so can't be assigned to. I could make a new local variable if you like, but I figured the ternary operator to be more clean.

Reply:
In this case, data.is_default_community is just a bool, so its always valid. Also it is a & non-mutable type, so can't be assigned to. I could make a new local variable if you like, but I figured the ternary operator to be more clean.

Reply:
Its about consistency, all form values are handled that way. And the field is_default_community needs to be an option on CreateCommunity, EditCommunity and CommunityForm. Otherwise it would have to be specified explicitly for every api call, and thats really not what we want.

Reply:
Okay i pushed a change for this. Let me know if that looks good to you; I wasn't 100% sure on whether it was what you wanted.